### PR TITLE
[Windows] Add Windows11SDK.22621 to Win19

### PR DESCRIPTION
**Pull request by erik-bershel**
# Description
Windows SDK 10.0.22621 added to Windows Server 2019.

#### Related issue:
#6137

## Virtual environments affected
- [x] Windows 2019
- [ ] Windows 2022
-
## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-VS.ps1 [hunk 1]
```diff
@@ -42,6 +42,12 @@ if (Test-IsWin19) {
 	$sdkFileName = "sdksetup14393.exe"
 	$argumentList = ("/q", "/norestart", "/ceip off", "/features OptionId.WindowsSoftwareDevelopmentKit")
 	Install-Binary -Url $sdkUrl -Name $sdkFileName -ArgumentList $argumentList
+
+	# Install Windows 11 SDK version 10.0.22621.0
```
Comment (al-cheb) [inline images/win/scripts/Installers/Install-VS.ps1:46]: Maybe we should bump SDK version in the Install-WDK.ps1 script, but we should check first.

Comment (erik-bershel) [reply images/win/scripts/Installers/Install-VS.ps1:46]: I thought about it, but I'm not sure. The question is what set of options we need in the new SDK. And what happens if we change the main version of SDK. Need to test.

**Comment by miketimofeev** [comment]
@al-cheb @erik-bershel does it make sense to add this SDK to windows-2022 as well?

**Comment by erik-bershel** [comment]
@miketimofeev that SDK is an option for VS2022 and already used in config.

File: images/win/scripts/Installers/Install-VS.ps1 [hunk 2]
```diff
@@ -42,6 +42,12 @@ if (Test-IsWin19) {
...
+    $sdkUrl = "https://go.microsoft.com/fwlink/p/?linkid=2196241"
+    $sdkFileName = "sdksetup22621.exe"
+    $argumentList = ("/q", "/norestart", "/ceip off", "/features OptionId.UWPManaged OptionId.UWPCPP OptionId.UWPLocalized OptionId.DesktopCPPx86 OptionId.DesktopCPPx64 OptionId.DesktopCPParm64")
+    Install-Binary -Url $sdkUrl -Name $sdkFileName -ArgumentList $argumentList
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Install-VS.ps1:50]: nit:
```suggestion
	# Install Windows 11 SDK version 10.0.22621.0
        $sdkUrl = "https://go.microsoft.com/fwlink/p/?linkid=2196241"
        $sdkFileName = "sdksetup22621.exe"
        $argumentList = ("/q", "/norestart", "/ceip off", "/features OptionId.UWPManaged OptionId.UWPCPP OptionId.UWPLocalized OptionId.DesktopCPPx86 OptionId.DesktopCPPx64 OptionId.DesktopCPParm64")
        Install-Binary -Url $sdkUrl -Name $sdkFileName -ArgumentList $argumentList
```

**Review by al-cheb — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by ddobranic — APPROVED** [review]

**Pull request merged**